Cargo.toml

```diff
@@ -1,15 +1,15 @@
 [package]
 name = "poseidon252"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
We should bump once the height padding is implemented IMO. 

Since this new version is incomplete until we don't have it.